<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>495</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Arenita ERP</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblUsuario">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>61</width>
      <height>18</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Nombre de usuario&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string>Usuario</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblNombreUsuario">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>119</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuario: </string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>495</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModulos">
    <property name="title">
     <string>MÃ³dulos</string>
    </property>
    <addaction name="actionAdministrador"/>
    <addaction name="separator"/>
    <addaction name="actionContabilidad"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionAcerca_de"/>
   </widget>
   <addaction name="menuModulos"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>BottomToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="separator"/>
  </widget>
  <action name="actionAcerca_de">
   <property name="text">
    <string>Acerca de</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionAdministrador">
   <property name="text">
    <string>Administrador</string>
   </property>
  </action>
  <action name="actionContabilidad">
   <property name="text">
    <string>Contabilidad</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
